### ISSUE TITLE
Bootstrapping compiler and interpreter don't support lambda forms

### ISSUE THREAD

**Issue by drmeister** [drmeister/clasp#110]
((lambda (x y) (+ x y)) 1 2)   --> error

**Comment by stassats** [comment]
That's the same as #70

**Comment by Shinmera** [comment]
Can confirm on b05884f.
```
> ((lambda (x y) (+ x y)) 1 2)

Condition of type: SIMPLE-ERROR
Handle codegen of cons: ((LAMBDA (X Y) (+ X Y)) 1 2)
```
The issue is not present on the cleavir version though.

**Comment by Shinmera** [comment]
Fixed on 712260b4038b40e04d8928b7183e2af5da416a0d
```
../../build/clasp/Contents/execs/boehm/release/bin/clasp -f bclasp
Starting boehm-0.3-test-10-480-g712260b ... loading image... it takes a few seconds
Top level.
> ((lambda (x y) (+ x y)) 1 2)

3
```
